src/main/java/com/commercetools/sync/sdk2/commons/utils/ResourceIdentifierUtils.java

```diff
@@ -0,0 +1,61 @@
+package com.commercetools.sync.sdk2.commons.utils;
+
+import static java.util.Optional.ofNullable;
+
+import com.commercetools.api.models.ResourceIdentifiable;
+import com.commercetools.api.models.common.ResourceIdentifier;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Objects;
+import java.util.Optional;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public final class ResourceIdentifierUtils {
+  public static final String REFERENCE_TYPE_ID_FIELD = "typeId";
+  public static final String REFERENCE_ID_FIELD = "id";
+
+  /**
+   * Given a {@link io.sphere.sdk.models.Referenceable} {@code resource} of the type {@code T}, if
```

Comment:
Please check all links in comments. Thx.
```suggestion
   * Given a {@link io.sphere.sdk.models.Referenceable} {@code resource} of the type {@code T}, if
```

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/932/commits/a97ab7e2dc854bdd2977cb149c8d83f7b206477d